app/scrapers/generalized.py

```diff
@@ -28,15 +27,12 @@ def get_page(self, query, startIndex=0):
         response = requests.get(self.url, headers=self.headers, params=payload)
         return response
 
-    @classmethod
     def parseResponse(self, soup):
         raise NotImplementedError
 
-    @classmethod
     def nextStart(self, currentStart, prevResults):
         return currentStart + len(prevResults)
 
-    @classmethod
```

Comment:
self is used in the method body so `@classmethod` must be removed.

Reply:
It is removed already
Or am I getting it wrong?